spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -2664,6 +2664,34 @@ public void testTableDescriptionRemainsUnchanged() {
         .isEqualTo(bq.getTable(testDataset.toString(), testTable).getDescription());
   }
 
+  @Test
+  public void testDirectWriteCountAfterWrite() throws InterruptedException {
```

Comment:
Why not to have a single `testCountAfterWrite()` ? do we need different logic for direct vs indirect writes?

Reply:
Right, missed that.